js/package.json

```diff
@@ -35,9 +38,16 @@
         "vinyl": "^3.0.0",
         "vinyl-paths": "^5.0.0"
     },
+    "files": [
+        "src/",
+        "bin/",
+        "slice/",
+        "README.md"
+    ],
     "scripts": {
         "build": "gulp",
         "dist": "gulp dist",
-        "clean": "gulp clean"
+        "clean": "gulp clean",
+        "prepack": "node scripts/prepack.js"
```

Comment:
The prepack script runs before "pack" and copies Slice files and Slice to JavaScript compilers to the required location.